src/utils/index.ts

```diff
@@ -1,4 +1,5 @@
 export { default as sysexits } from './sysexits';
+export { default as Plug } from './Plug';
```

Comment:
Can we remove this, and rely on the `Lock` instead?

Reply:
That's using the lock. Getting the lock to work as a plug properly needed some extra logic that I would've had to copy in 4 places. So I made a utility class to handle it.

Reply:
Resolving this, discussion for it in the review comment below.